doc/features/ffmpeg/README.md

```diff
@@ -0,0 +1,9 @@
+# ffmpeg-qsv capabilities
+
+* [AV1](./av1.md)
+* [H264](./h264.md)
+* [H265](./h265.md)
```

Comment:
H.265 or better yet rename to H.265/HEVC

Reply:
done